staging/traefik-forward-auth/templates/hooks.yaml

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
Since this is a staging chart, and we generally intend to upstream this do we (or don't we) want some of this functionality in this chart to be conditional rather than deploying the jobs by default?

Reply:
What is this password and secret used for? Do we expect users wanting to generate it?